nrf-hal-common/src/saadc.rs

```diff
@@ -23,6 +39,9 @@ pub use saadc::{
 // multiple channels should work (See "scan mode" in the datasheet).
 // Issue: https://github.com/nrf-rs/nrf-hal/issues/82
 
+#[doc = "Interface for the SAADC peripheral.
```

Comment:
You can use the `#[doc]` attribute, though this is typically only used in cases like macros where expansion doesn't work. Instead, you can use use `///`, e.g.

```rust
/// This is a doc comment
/// It can be multiple lines!
pub struct Saadc(SAADC);
```